src/business_logic/fact_state/state.rs

```diff
@@ -234,19 +228,19 @@ pub(crate) struct StateDiff {
     address_to_class_hash: HashMap<Address, [u8; 32]>,
     address_to_nonce: HashMap<Address, Felt>,
     storage_updates: HashMap<Felt, HashMap<[u8; 32], Address>>,
-    block_info: BlockInfo,
 }
 
-impl StateDiff {
-    pub fn empty(block_info: BlockInfo) -> Self {
+impl Default for StateDiff {
```

Comment:
We can just derive this in the struct instead of adding the method with `#[derive(Default)]` in the struct definition.